addons/resource/views/resource_calendar_views.xml

```diff
@@ -50,17 +50,26 @@
                     <div class="oe_title">
                         <h1><field name="name"/></h1>
                     </div>
-                    <group name="resource_details">
-                        <group>
+                    <group name="resource_details" col="2">
+                        <group name="resource_working_hours">
+                            <field name="flexible_hours"/>
+                            <label for="full_time_required_hours" invisible="flexible_hours"/>
+                            <label for="full_time_required_hours" string="Hours per Week" invisible="not flexible_hours"/>
+                            <div>
+                                <field name="full_time_required_hours" style="width: 10%" widget="float_time"/>
```

Comment:
Setting `class=w-10` added an unwanted indentation (same result when removing the style defined here), so I'm not sure this will work :grimacing:
![image](https://github.com/user-attachments/assets/8ee86c1c-8416-499f-b692-04be3f3ceef1)